docs/operations/github_ci_app.rst

```diff
@@ -0,0 +1,73 @@
+######################################
+Adding a new GitHub CI app integration
+######################################
+
+
+Create a new GitHub app
+=======================
+
+#. Click the ``New GitHub App`` button in the `lsst-sqre org Developer Settings apps page <https://github.com/organizations/lsst-sqre/settings/apps>`__.
+#. Name it ``mobu CI (<env URL or id if the URL is too long>)``.
+#. Make sure the ``Active`` checkbox is checked in the ``Webhook`` section.
+#. Enter ``https://<env URL>/mobu/github/ci/webhook`` in the ``Webhook URL`` input.
+#. Generate a strong password to use as the webhook secret.
+#. Store this in the ``SQuaRE`` vault in the ``LSST IT`` 1Password account in an item named ``mobu GitHub CI app webhook secret (<env URL>)``.
```

Comment:
As an aside, I've started making a single 1Password entry for stuff like this with key/value pairs for each secret. The `Server` type is sometimes convenient for this since you can delete all of the structure and then make up whatever structure in 1Password works the best. See how the Vault tokens are stored in 1Password, for example.